releasenotes/notes/Add-Group-exec-perm-for-the-SecretBackend-command-a7fbe1ece1fad50b.yaml

```diff
@@ -0,0 +1,7 @@
+---
+enhancements:
+  - |
+    Add new configuration parameter to allow 'GroupExec' permission on the secret-backend command.
```

Comment:
Could we develop this to quote the new parameter name and explain in which case it should be enabled ?